src/main/java/io/phasetwo/service/resource/InvitationsResource.java

```diff
@@ -83,7 +83,9 @@ public Response createInvitation(@Valid InvitationRequest invitation) {
     }
 
     try {
-      InvitationModel i = organization.addInvitation(email, auth.getUser());
+      UserModel inviter = session.users().getUserById(realm, invitation.getInviterId());
```

Comment:
For backwards compatibility, I'd like to have it fall back to `auth.getUser()` if the inviterId is null/empty.

Reply:
@xgp thx for review - updated the code in question. LMK if it's all right or there are better ways - my Java is a bit rusty unfortunately :)